.github/ISSUE_TEMPLATE/1-bug-report.yaml

```diff
@@ -0,0 +1,78 @@
+name: "\U0001F41B Bug report"
+description: Create a report to help Matter
+title: "[BUG] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting an issue against the Matter SDK! We need information about the bug report to follow up, so please help us out by filling out this information.
+  - type: textarea
+    id: repro
+    attributes:
+      label: Reproduction steps
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: bash
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week, day, hour)"
```

Comment:
Can we have a matter-specific example instead of "personal account page"?

Reply:
Fixed